blog/2022-07-22-centralized-kubectl-access/index.md

```diff
@@ -0,0 +1,84 @@
+---
+slug: centralized-kubectl-access
+title: "Configuring Centralized Kubectl Access To Clusters With Paralus"
+authors: [atul]
+tags: [tutorial, paralus]
+---
+
+Kubectl is one of the most widely used tools to work with Kubernetes. The command line tool allows you to deploy applications, inspect and manage resources. It basically authenticates with the control plane for your cluster and makes API calls to the Kubernetes API. In short if you are working with Kubernetes - you will use kubectl the most.
```

Comment:
[optional] As we are talking about kubectl for newbies, how about we add a reference link, https://kubernetes.io/docs/reference/kubectl/? Just a hyperlink to Kubectl text will work.